pop/app.go

```diff
@@ -0,0 +1,353 @@
+/*
+* This is a template for creating an app. It only has one command which
+* prints out the name of the app.
+ */
+package main
+
+import (
+	"os"
+
+	"io/ioutil"
+
+	"path"
+
+	"errors"
+
+	"encoding/base64"
+
+	"bytes"
+
+	"net"
+
+	"fmt"
+
+	"strings"
+
+	"github.com/BurntSushi/toml"
+	"github.com/dedis/cothority/cosi/check"
+	"github.com/dedis/cothority/pop/service"
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/anon"
+	"github.com/dedis/crypto/random"
+	"gopkg.in/dedis/onet.v1/app"
+	"gopkg.in/dedis/onet.v1/log"
+	"gopkg.in/dedis/onet.v1/network"
+	"gopkg.in/urfave/cli.v1"
+)
+
+func init() {
+	network.RegisterMessage(Config{})
+}
+
+var client *service.Client
+
+// Config represents either a manager or an attendee configuration.
+type Config struct {
+	Private abstract.Scalar
+	Public  abstract.Point
+	Index   int
+	Address network.Address
+	Final   *service.FinalStatement
+}
```

Comment:
Can you give a very short comment about what is each field ? for example, I've no idea what Index means nor Address.